Gruntfile.js

```diff
@@ -26,6 +27,103 @@ module.exports = function (grunt) {
+                        'CELLBASE_HOST': 'wwwint.ebi.ac.uk/cellbase/webservices/rest',
+                        'CELLBASE_VERSION': 'v3'
+                    }
+                }
+            },
+            dev: {
+                options: {
+                    variables: {
+                        'METADATA_HOST': 'wwwint.ebi.ac.uk/eva/webservices/rest',
+                        'METADATA_VERSION': 'v1',
+                        'CELLBASE_HOST': 'wwwint.ebi.ac.uk/cellbase/webservices/rest',
+                        'CELLBASE_VERSION': 'v3'
+                    }
+                }
+            },
+            stage: {
+                options: {
+                    variables: {
+                        'METADATA_HOST': 'wwwdev.ebi.ac.uk/eva/webservices/rest',
+                        'METADATA_VERSION': 'v1',
+                        'CELLBASE_HOST': 'wwwdev.ebi.ac.uk/cellbase/webservices/rest',
+                        'CELLBASE_VERSION': 'v3'
+                    }
+                }
+            },
+            prod: {
+                options: {
+                    variables: {
+                        'METADATA_HOST': 'www.ebi.ac.uk/eva/webservices/rest',
+                        'METADATA_VERSION': 'v1',
+                        'CELLBASE_HOST': 'www.ebi.ac.uk/cellbase/webservices/rest',
+                        'CELLBASE_VERSION': 'v3'
+                    }
+                }
+            }
+        },
+        replace: {
+            eva_manager: {
+                options: {
+                    patterns: [
+                        {
+                            match: 'METADATA_HOST',
+                            replacement: '<%= grunt.config.get("METADATA_HOST") %>'
+                        },
+                        {
+                            match: 'METADATA_VERSION',
+                            replacement: '<%= grunt.config.get("METADATA_VERSION") %>'
+                        },
+                        {
+                            match: 'CELLBASE_HOST',
+                            replacement: '<%= grunt.config.get("CELLBASE_HOST") %>'
+                        },
+                        {
+                            match: 'CELLBASE_VERSION',
+                            replacement: '<%= grunt.config.get("CELLBASE_VERSION") %>'
+                        }
+                    ]
+                },
+                src: 'src/js/eva-manager-config.js',
+                dest: 'src/js/eva-manager.js'
+            },
+            acceptance_test: {
+                options: {
+                    patterns: [
+                        {
+                            match: 'BASE_URL',
+                            replacement: 'http://localhost/eva-web/build/<%= meta.version.eva %>/index.html'
+                        }
+                    ]
+                },
+                src: 'tests/mocha/acceptance/config-manager.js',
+                dest: 'tests/mocha/acceptance/config.js'
```

Comment:
I guess tests files could be moved under `tests` instead of `tests/mocha`? I don't see any files in the intermediate folder.